### ISSUE TITLE
Failed to activate the atom-material-ui theme

### ISSUE THREAD

**Issue by Shadownigma** [atom-material/atom-material-ui#354]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.14.2 x64
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Enterprise
**Thrown From**: [atom-material-ui](https://github.com/silvestreh/atom-material-ui) package 1.3.9

### Stack Trace

Failed to activate the atom-material-ui theme

```
At missing opening `(` in C:\Users\gomesa3\.atom\packages\atom-material-ui\styles\custom.less:2:0

LessError: missing opening `(`
  at /packages/atom-material-ui/styles/custom.less:2:0
```

### Commands

```
```

### Non-Core Packages

```
atom-bootstrap4 1.4.0 
atom-material-syntax-dark 0.2.7 
atom-material-ui 1.3.9 
emmet 2.4.3 
file-icons 2.0.15 
minimap 4.26.8 
monokai 0.20.0 
pigments 0.39.0 
script 3.14.1 
sort-lines 0.14.0 
```

**Comment by silvestreh** [comment]
This is fixed in Material UI `2.0`. It's available for Atom 1.17-beta users!

**Comment by kentico-anthonym** [comment]
It seems this was caused by an empty new line at the end of the file. For anybody that is currently having this issue.
